network/config.go

```diff
@@ -85,12 +83,18 @@ type Config struct {
 	Name string `json:"name"`
 	// Backend specifies the backend for the network
 	Backend Backend `json:"backend"`
-	// Flags can hold additional flags for avalanchego nodes.
+	// Flags that will be passed to each node in this network.
 	// It can be empty.
-	// The precedence of flags handling is:
-	// 1. Flags defined in node.Config override
-	// 2. Flags defined in network.Config (this struct) override
-	// 3. Flags defined in the json config file
+	// Config flags may also be passed in a node's config struct
+	// or config file.
+	// The precedence of flags handling is, from highest to lowest:
+	// 1. Flags defined in a node's node.Config
+	// 2. Flags defined in a network's network.Config
+	// 3. Flags defined in a node's config file
+	// For example, if a network.Config has flag W set to X,
+	// and a node within that network has flag W set to Y,
+	// and the node's config file has flag W set to Z,
+	// then the node will be started with flag W set to X.
```

Comment:
It would be set to `Y` - unless I don't understand line 95?

Reply:
woops! fixed, good catch.